apps/web/src/components/Composer/Actions/Attachment.tsx

```diff
@@ -60,14 +47,27 @@ const Attachment: FC = () => {
     return false;
   };
 
-  const isImageType = (files: FileList) => {
-    for (const file of files) {
-      if (!ALLOWED_IMAGE_TYPES.includes(file.type)) {
-        return false;
-      }
+  const mediaLimitReached = (files: FileList) => {
+    //gets type of media through first 5 letters of type (media || video || audio)
+    const mediaType = files[0].type.substring(0, 5);
+
+    //If image limit reach toggle flag to disabled additional uploads
+    switch (mediaType) {
+      case 'image':
+        setIsImageAttachmentType(true);
+        break;
+      case 'audio':
+        toggleLimitReached(true);
+        break;
+      case 'video':
```

Comment:
You can combine to single condition here

Reply:
Combined